client/container.go

```diff
@@ -816,3 +816,23 @@ func (c *Client) ContainerAnnounceUsedSpace(ctx context.Context, prm PrmAnnounce
 
 	return &res, nil
 }
+
+// SyncContainerWithNetwork requests network configuration using passed client
+// and applies it to the container.
+//
+// Note: if container does not match network configuration, SyncContainerWithNetwork
+// changes it.
+//
+// Returns any network/parsing config errors.
+//
+// See also NetworkInfo, container.ApplyNetworkConfig.
+func SyncContainerWithNetwork(ctx context.Context, cnr *container.Container, c *Client) error {
```

Comment:
Lets add similar function to `pool` package, it'd be useful in services like S3 gateway https://github.com/nspcc-dev/neofs-s3-gw/blob/5bfc54974669bfddf1e9693d9625578a8167778f/internal/neofs/neofs.go#L137.

Reply:
/cc @alexvanin, @KirillovDenis